helpers/_mixins.scss

```diff
@@ -38,3 +38,40 @@
         @content;
     }
 }
+
+//===============================================================//
+// Customize pre-existing animations and create one-offs         //
+//===============================================================//
+//      .fadeInSlow {                                            //
+//          @include animate-custom(fadeIn, 4s);                 //
+//      }                                                        //
+//===============================================================//
+// Courtesy of @aviladotandres: (twitter.com)                    //
+//===============================================================//
+@mixin animate-custom( $name,
+        $duration:$base-duration,
```

Comment:
I think that this should be interpolated in use $base-duration is set using a CSS custom property.

e.g: `$duration:#{$base-duration},`